docs/style.css

```diff
@@ -0,0 +1,557 @@
+    background: var(--light);
+}
+
+.playstore-btn svg:hover {
+    fill: var(--light);
+}
+
+.appstore-btn svg:hover {
+    fill: var(--light);
+}
+
+.apple-store svg:hover {
+    stroke: var(--light);
+}
+
+.playstore-btn svg {
+    fill: var(--dark);
+}
+
+.appstore-btn svg {
+    fill: var(--dark);
+    stroke: var(--dark);
+}
+
+.appstore-btn:hover, .playstore-btn:hover {
+    background: var(--container);
+}
+
+.images {
+    display: flex;
+    gap: 40px;
+}
+
+.column1, .column2 {
+    display: grid;
+    gap: 40px;
+}
+
+.about-picture {
+    border-radius: 20px;
+}
+
+.about-picture.picture-1 {
+    width: 660px;
+    height: 590px;
+}
+
+.about-picture.picture-2 {
+    width: 660px;
+    height: 430px;
+}
+
+.about-picture.picture-3 {
+    width: 100%;
+    height: 430px;
+}
+
+.about-picture.picture-4 {
+    width: 660px;
+    height: 590px;
+}
+
+.mobile-app.container {
+    display: flex;
+    gap: 100px;
+}
+
+.menu {
+    display: flex;
+    flex-direction: column;
+    margin-top: -10px;
+}
+
+.menu.container {
+    height: 1328px;
+    text-align: center;
+}
+
+.menu-offer {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 0px;
+}
+
+.menu-grid {
+    margin-top: 30px;
+}
+
+.name-and-the-price-of-the-coffee {
+    padding-left: 15px ;
+}
+
+.coffee-description {
+    margin-top: -15px;
+    padding: 0 10px 0 15px;
+}
+
+.tab-list {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 16px;
+    justify-content: space-between;
+}
+
+.item{
+    display: flex;
+    align-items: center;
+    border: 1px solid #C1B6AD;
+    gap: 8px;
+    border-radius: 100px;
+    width: fit-content;
+    height: 46px;
+    background: var(--light);
+    padding: 8px 16px 8px 8px;
+    transition: .5s;
+}
+
+.icon {
+    border-radius: 100px;
+    background: #C1B6AD;
+    width: 30px;
+}
+
+.item:hover {
+    background-color: var(--container);
+    color: var(--light);
+}
+
+.item:hover .icon {
+    background: #E1D4C9;
+}
+
+.item1:active .icon {
+    background: #E1D4C9;
+}
+
+.preview {
+    flex-wrap: wrap;
+    border: 1px solid #C1B6AD;
+    transition: .25s ease-in;
+    width: 310px;
+    border-radius: 40px;
+    text-align: left; 
+}
+
+.preview:hover .preview {
+    width: 279px;
+    height: 279px;
+    cursor: pointer;
+}
+
+.coffee-card {
+    display: flex;
+}
+
+
+.menu-grid {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 37px;
+    padding-bottom: 100px;
+}
+
+.mobile-app.container {
+    text-align: left;
+}
+
+.contact-link {
+    text-decoration: none;
+    color: var(--light);
+}
+
+.contact-link:hover {
+    text-decoration: underline var(--light);
+}
+
+.social-buttons:hover {
+    cursor: pointer;
+    fill: #E1D4C9;
+}
+
+.social-buttons:hover .social-btn-elements {
+    fill: var(--dark);
+}
+
+.download-text {
+    padding-top: 100px;
+}
+
+.Footer-Text1 {
```

Comment:
Please rename the classes to fit to your convention

Reply:
Done.